app/build.gradle

```diff
@@ -123,7 +123,7 @@ dependencies {
 
     // Firebase
     // Adjust version for emulator
-    def firebaseVersion = '11.6.2'
+    def firebaseVersion = '11.8.0'
```

Comment:
Can you revert this?
because the latest firebase version doesn't work well on the emulator.

Reply:
Oops. I'll resolve it soon...